src/planwise/client/projects/handlers.cljs

```diff
@@ -25,8 +25,9 @@
  :projects/create-project
  in-projects
  (fn [db [_ project-data]]
-   (async-handle (api/create-project project-data)
-                 #(dispatch [:projects/project-created %]))
+   (api/create-project
+     project-data
+     #(dispatch [:projects/project-created %]))
```

Comment:
If we turn `json-opts` to a function, we could allow passing a `:keyword` as `success-fn`, which is understood as dispatching that event using the response as data. So the above block would be:
```clojure
(api/create-project project-data :projects/project-created)
```